coremltools/converters/mil/mil/ops/defs/recurrent.py

```diff
@@ -9,6 +9,80 @@
 
 @register_op(doc_str="TODO")
 class gru(Operation):
+    """
+    Gated recurrent unit (GRU).
+    
+    .. math::
+       r_t = recurrent_activation(W_{ir} x_t + b_{ir} + W_{hr} h_(t−1) + b_{hr})
```

Comment:
These equations are not rendered properly. It should be I guess `r_t = \rm{recurrent\_activation}(W_{ir} x_t + b_{ir} + W_{hr} h_{t-1} + b_{hr})`